trade-finance-manager-ui/templates/utilisation-reports/_macros/premium-payments-table.njk

```diff
@@ -3,8 +3,109 @@
 {% import "../../_macros/table-cell-checkbox.njk" as tableCellCheckbox %}
 {% import "../../_macros/select-all-table-cell-checkbox.njk" as selectAllTableCellCheckbox %}
 
+{% macro renderPremiumPaymentsTableRow(params) %}
```

Comment:
Can I ask that we put this macro in it's own file? It's fine for the tests to just cover the outer macro, so no need to refactor them as I don't think we would use this elsewhere but it's just nice to keep the nunjucks files short and as readable as possible.